distribution/lib/Standard/Database/0.0.0-dev/src/Data/Column.enso

```diff
@@ -1133,10 +1181,26 @@ type Column
 
              column.replace '"(.*?)"'.to_regex '($1)'
     replace : Text | Column | Regex -> Text | Column -> Case_Sensitivity -> Boolean -> Column
-    replace self term="" new_text="" case_sensitivity=Case_Sensitivity.Sensitive only_first=False =
-        _ = [term, new_text, case_sensitivity, only_first]
-        msg = "`Column.replace` is not yet implemented."
-        Error.throw (Unsupported_Database_Operation.Error msg)
+    replace self term="" new_text="" case_sensitivity=Case_Sensitivity.Default only_first=False =
+        case_sensitivity.disallow_non_default_locale <|
+            Value_Type.expect_text self <|
+                term_name = case term of
+                    _ : Text -> term
+                    _ : Column -> term
+                    _ : Regex -> term.pattern_string
+                new_name = self.naming_helpers.function_name "replace" [self, term_name, new_text]
+                use_regex_part = case term of
+                    _ : Text -> "TEXT"
+                    _ : Column -> "COLUMN"
+                    _ : Regex -> "REGEX"
+                case_sensitivity_part = case case_sensitivity of
+                    Case_Sensitivity.Default -> "CS"
```

Comment:
IMO `Case_Sensitivity.Default` should map to a default CS. Does it not depend on the collation associated with a given column? Ideally, we should rely on the default sensitivity in this case. If possible. If not possible, then this looks OK, but I'd consider adding some note somewhere :)

Reply:
This logic is now moved to the dialects, where we can be dialect-specific.